src/openrct2/network/NetworkBase.cpp

```diff
@@ -2894,7 +2894,7 @@ void NetworkBase::Server_Handle_CHAT(NetworkConnection& connection, NetworkPacke
 void NetworkBase::Client_Handle_GAME_ACTION([[maybe_unused]] NetworkConnection& connection, NetworkPacket& packet)
 {
     uint32_t tick;
-    uint32_t actionType;
+    GameCommand actionType;
```

Comment:
Ditto for these `GameCommand actionType`s - check nothing else can be received here.

Reply:
Same for this.